### PR TITLE
Update: added steps, muted keys/id File: auditing-streaming.md

### PR DESCRIPTION
**Pull request by dylan-apera**
Update 1: Added additional step 3 line 114 to include the expansion of a tab in azure Log Analytics Workspace to reflect the new view in Azure portal. 

Update 2: Muted/hid the workspace ID and keys.

ACTION REQUIRED: I need to update the .png file that is on line 117 and replace with new one.

azure-monitor-log-keys-update

### CODE REVIEW

**Comment by Court72** [comment]
@chcomley

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/organizations/audit/auditing-streaming.md [hunk 1]
```diff
@@ -110,15 +110,16 @@ Once you have your Event Grid stream configured, you can set up subscriptions on
 ### Set up an Azure Monitor Log stream
 
 1. Create a [Log Analytics workspace](/azure/azure-monitor/learn/quick-create-workspace).
-2. Open the workspace and select **Agents management**.
-3. Make note of the workspace ID and primary key.
+2. Open the workspace and select **Agents**.
+3. Extend Log Analytics agent instructions
```
Comment (chcomley) [inline docs/organizations/audit/auditing-streaming.md:114]: I don't understand this step, please clarify, and then add a period at the end of the sentence. Thanks!

**Comment by chcomley** [comment]
azure-monitor-log-keys-update
Here's a new screenshot, if this works...

File: docs/organizations/audit/auditing-streaming.md (hunk 1 above)
Comment (dylan-apera) [reply docs/organizations/audit/auditing-streaming.md:114]: Hi @chcomley, 114 step 3. When you open LAW the page is not visible with the keys and or Workspace ID. So I guess I can change that to: "Click on Log Analytics agent instructions drop down to open and view Workspace ID and Primary Key."

**Comment by chcomley** [comment]
#sign-off

File: docs/organizations/audit/auditing-streaming.md [hunk 2]
```diff
@@ -110,15 +110,16 @@ Once you have your Event Grid stream configured, you can set up subscriptions on
 ### Set up an Azure Monitor Log stream
 
 1. Create a [Log Analytics workspace](/azure/azure-monitor/learn/quick-create-workspace).
-2. Open the workspace and select **Agents management**.
-3. Make note of the workspace ID and primary key.
+2. Open the workspace and select **Agents**.
+3. Select the Log Analytics agent instructions dropdown menu to open and view the workspace ID and primary key.
```
Comment (ShannonLeavitt) [inline docs/organizations/audit/auditing-streaming.md:114]:
```suggestion
3. Select **Log Analytics agent instructions** to view the workspace ID and primary key.
```
Updated sentence according to UI shown in the PR's conversation view.

**Pull request merged**